### ISSUE TITLE
High verbosity with CBC MILP Solver

### ISSUE THREAD

**Issue by epruesse** [snakemake/snakemake#631]
**Snakemake version** 

5.24.0

**Describe the bug**

A message from the CBC MILP Solver is printed with every job execution.

**Logs**
```
Welcome to the CBC MILP Solver 
Version: 2.10.5 
Build Date: Apr  7 2020 

command line - cbc /tmp/b408874a7b664f0581980b4810bbd5bc-pulp.mps max ratio None allow None threads None presolve on strong None gomory on knapsack on probing on branch printingOptions all solution /tmp/b408874a7b664f0581980b4810bbd5bc-pulp.sol (default strategy 1)
At line 2 NAME          MODEL
At line 3 ROWS
At line 7 COLUMNS
At line 28 RHS
At line 31 BOUNDS
At line 36 ENDATA
Problem MODEL has 2 rows, 4 columns and 8 elements
Coin0008I MODEL read with 0 errors
String of None is illegal for double parameter ratioGap value remains 0
String of None is illegal for double parameter allowableGap value remains 0
String of None is illegal for integer parameter threads value remains 0
String of None is illegal for integer parameter strongBranching value remains 5
Option for gomoryCuts changed from ifmove to on
Option for knapsackCuts changed from ifmove to on
Continuous objective value is 2 - 0.00 seconds
Cgl0004I processed model has 0 rows, 0 columns (0 integer (0 of which binary)) and 0 elements
Cbc3007W No integer variables - nothing to do
Cuts at root node changed objective from -2 to -1.79769e+308
Probing was tried 0 times and created 0 cuts of which 0 were active after adding rounds of cuts (0.000 seconds)
Gomory was tried 0 times and created 0 cuts of which 0 were active after adding rounds of cuts (0.000 seconds)
Knapsack was tried 0 times and created 0 cuts of which 0 were active after adding rounds of cuts (0.000 seconds)
Clique was tried 0 times and created 0 cuts of which 0 were active after adding rounds of cuts (0.000 seconds)
MixedIntegerRounding2 was tried 0 times and created 0 cuts of which 0 were active after adding rounds of cuts (0.000 seconds)
FlowCover was tried 0 times and created 0 cuts of which 0 were active after adding rounds of cuts (0.000 seconds)
TwoMirCuts was tried 0 times and created 0 cuts of which 0 were active after adding rounds of cuts (0.000 seconds)
ZeroHalf was tried 0 times and created 0 cuts of which 0 were active after adding rounds of cuts (0.000 seconds)

Result - Optimal solution found

Objective value:                2.00000000
Enumerated nodes:               0
Total iterations:               0
Time (CPU seconds):             0.02
Time (Wallclock seconds):       0.00

Option for printingOptions changed from normal to all
Total time (CPU seconds):       0.02   (Wallclock seconds):       0.00
```

**Additional context**

See #603 -- this seems to be a similar issue.

**Comment by bkohrn** [comment]
I actually opened a separate issue for this before I realized this issue already existed (#637), but I'd like to second this, except I see it printing out multiple times per workflow.  My workflow is mostly based on shell blocks.

**Comment by FelixMoelder** [comment]
This should have been fixed in snakemake 5.25.0. Does this issue still exist?

**Comment by bkohrn** [comment]
It seems to be fixed for my pipeline once I updated to 5.25.0.  Thank you!
